pkg/virt-controller/watch/vm.go

```diff
@@ -2629,7 +2629,10 @@ func (c *VMController) sync(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachin
 			vmCopy.Spec.Template.Spec.Domain.Devices.Interfaces = ifaces
 			vmCopy.Spec.Template.Spec.Networks = networks
 
-			handleDynamicInterfaceRequests(vmCopy)
+			err := handleDynamicInterfaceRequests(vmCopy, vmiHasPodNetwork(vmi), c.clusterConfig)
+			if err != nil {
+				syncErr = &syncErrorImpl{fmt.Errorf("Error encountered while handling interface hotplug requests: %v", err), HotPlugVolumeErrorReason}
```

Comment:
`HotPlugVolumeErrorReason` seems to be the wrong error here.

Reply:
Done.